application/visualization/duration_breakdown.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
```

Comment:
In the next iteration it might make sense to put these "breakdown" files into one module, there is a lot of shared code. However we might not touch these again, so it might not be neccessary.